.devcontainer/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
 
     environment:
       - "ACCEPT_EULA=Y"
-      - "SA_PASSWORD=yourStrong(!)Password"
+      - "MSSQL_SA_PASSWORD=QhNemHjCWHui9vNnRC.TMRAhusvM3n9_"
```

Comment:
May I ask is this password here generated in someway during the run time?

Reply:
It is not generated, I just regenerated the password to reduce reliance on hardcoded non-random passwords. 
It doesn't change any behavior.

I did update the environmental variable name as the one that was used is no longer supported.